src/Nox.Generator/Application/Factories/EntityLocalizedFactory.template.cs

```diff
@@ -33,4 +33,21 @@ internal abstract class {{className}}Base : IEntityLocalizedFactory<{{localizedE
 
         return localizedEntity;
     }
+
+    public virtual void UpdateLocalizedEntity({{ localizedEntityName}} localizedEntity, {{entity.Name}}UpdateDto updateDto, CultureCode cultureCode)
```

Comment:
seems that the culture code is unnecessary here

Reply:
You are right. I will fix that in the PartialUpdate PR that I'm working on now.